.github/action-common-python-release/action.yml

```diff
@@ -0,0 +1,76 @@
+name: "Base Python Release"
+description: "Common tasks required to release python wheels on any platform"
+
+inputs:
+  package-name:
+    required: true
+    description: "Either `qcs-sdk-python` or `qcs-sdk-python-grpc-web`"
+
+  python-version:
+    default: "3.12"
+    description: "Python version used for script steps"
+  python-architecture:
+    description: "Python architecture used for script steps"
+  rust-target:
+    description: "Rust target used for script steps"
+  maturin-command:
+    default: build
+    description: "Maturin command used, `build` or `sdist`"
+  maturin-args:
+    description: "Additional maturin command arguments"
+  maturin-target:
+    description: "Rust target used for maturin compilation"
+
+
+runs:
+  using: "composite"
+  steps:
+  - uses: actions/setup-python@v4
+    with:
+      python-version: ${{ inputs.python-version }}
+      architecture: ${{ inputs.python-architecture }} 
+  - name: Install Rust toolchain 
+    uses: actions-rs/toolchain@v1 
+    with: 
+      toolchain: stable 
+      target: ${{ inputs.rust-target }}
+      profile: minimal 
+      default: true 
+  - name: Patch package metadata for grpc-web
+    shell: bash
+    if: inputs.package-name == 'qcs-sdk-python-grpc-web'
+    run: |
+      pip install toml
+      python crates/python/scripts/patch_grpc_web.py
+      cargo update hyper-proxy
+
+  # must be build or sdist
+  - if: inputs.maturin-command == 'build'
+    name: Maturin - Build
+    uses: messense/maturin-action@v1
+    with:
+      manylinux: '2_28'
+      command: ${{ inputs.maturin-command }}
+      target: ${{ inputs.maturin-target }}
+      args: -i 3.8 3.9 3.10 3.11 3.12 --release --manifest-path crates/python/Cargo.toml --out dist ${{ inputs.package-name == 'qcs-sdk-python-grpc-web' && '-F grpc-web' || '' }}
+      docker-options: -e CI
+  - if: inputs.maturin-command == 'sdist'
+    name: Maturin - Source Distribution
+    uses: messense/maturin-action@v1
+    with:
+      manylinux: '2_28'
+      command: ${{ inputs.maturin-command }}
+      target: ${{ inputs.maturin-target }}
+      args: --manifest-path crates/python/Cargo.toml --out dist
+
+  # host will not produce an installable wheel in this case
```

Comment:
Just for clarity, the issue is that the wheel isn't installable on the host because it's for a different architecture, not that the wheel isn't installable.

Reply:
Correct. I was able to verify that an installation of the wheel from https://github.com/rigetti/qcs-sdk-rust/actions/runs/8548321434 produced a valid and usable python dep.

Reply:
```suggestion
  # host will not be able to install wheel in this case
```